feign/src/main/java/com/baeldung/core/exception/NotFoundException.java

```diff
@@ -0,0 +1,18 @@
+package com.baeldung.core.exception;
+
+public class NotFoundException extends Exception {
+
```

Comment:
why we need these two exception classes?

Reply:
Because it is used here:
![image](https://user-images.githubusercontent.com/12120641/220111594-65a68ac1-8e47-48cd-a68b-67a002e8a3cc.png)